### ISSUE TITLE
Output Website dose not contain notes

### ISSUE THREAD

**Issue by CatWithCode** [yoursamlan/pubsidian#9]
I made a test vault and created one file. Both just have "abc 123" in them.
After i run the converter, the exported website dose not show my notes.
Are there some version dependencies with obsidian or do i miss something?
At the end i wanted the "Nokia.md" inside of the exported website. No imgs or anything
![image](https://user-images.githubusercontent.com/56090976/159136373-db6039f2-60b1-4330-86b0-267c56393913.png)

![image](https://user-images.githubusercontent.com/56090976/159136387-39923443-9f8d-4323-821a-e8f4533dd270.png)

(The Nokia.html is in the pages folder!)
![image](https://user-images.githubusercontent.com/56090976/159136401-41600545-1766-4482-8079-8e1c69909975.png)

![image](https://user-images.githubusercontent.com/56090976/159136433-7d83a637-fa2d-43bc-b6a5-fc0b014707c3.png)

**Comment by yoursamlan** [comment]
Hi, Sorry for the inconvenience caused to you.

Can you please check the `JS console log` by inspecting the element of the page? 
I think the main reason behind this issue is the blockage of `makelist()` function that generates the note list from `data.json`. If possible, please upload the code temporarily on a Github repo and send me the link, it will help me to debug the issue. 

Also,  if you have any problem, can you please paste the code of the `data.json`  here.

Thank you.

**Comment by CatWithCode** [comment]
Here is the hole output folder
[test.zip](https://github.com/yoursamlan/pubsidian/files/8312320/test.zip)
.

**Comment by CatWithCode** [comment]
Here is everything. Its only a few things:

Here is the data.json:
{
  "graph": [],
  "links": [
],
  "nodes": [
	{"sl":0, "size": 10, "score": 0.5, "id": "Nokia", "type":"circle"}
],
  "directed": false,
  "multigraph": false
}

Here the index.html:
<html><head><meta name='viewport' content='width=device-width, initial-scale=1'><link rel='icon' type='image/png' href='https://github.com/yoursamlan/pubsidian/blob/main/CDN/favicon.png?raw=true' /><title>pubsidian</title><link href='https://fonts.googleapis.com/css?family=Comfortaa' rel='stylesheet'><link href='https://fonts.googleapis.com/css?family=Montserrat' rel='stylesheet'><a href='https://github.com/yoursamlan' target='_blank' pubsidian id='init'></a><link rel='stylesheet' href='https://rawcdn.githack.com/yoursamlan/pubsidian/d380fd11ac87a9d58b325f1fa85892d0cdaacfc2/CDN/style.css' type='text/css'><script type='text/x-mathjax-config'>MathJax.Hub.Config({tex2jax: {inlineMath: [['$','$'], ['\(','\)']]}});</script></head><body><div id='wrapper'><div id='mySidebar' class='sidebar'><a href='javascript:void()' class='closebtn' onclick='closeNav()'>×</a><div id='content'><div id='sideheader'><h4 id='headtext'>MENU </h4></div><input type='text' id='mySearch' onkeyup='myFunction()' placeholder='Search..' title='Type in a category'><ul id='myMenu'></ul><div class='debugdiv'><h4>Dark mode</h4><div id='dbgbtn'><label class='switch'><input type='checkbox' id='dbg'><span class='slider' onclick='togdarkmode()'></span></label></div></div></div></div><div id='pagecontent'><div id='heading'><button class='openbtn' id='hamb' onclick='openNav()'>☰</button><div id='title'><h5>ppp's notebook</h5></div><button class='openbtn' id='graphbtn' onclick='openGraph();setTimeout(showGraph(), 400);'>&#128376;</button></div><div id='main'><h1 id='ContentHeading'>Welcome to Pubsidian<hr></h1><div id='mainContent'><img src='https://raw.githubusercontent.com/yoursamlan/pubsidian/main/CDN/pubsidian.png' alt='logo' style='height:10%; width:auto; display:block; margin-right:auto;float:none !important;'><p>Version: 3.2.0-beta <br>Developed by: <a href='https://github.com/yoursamlan' target='_blank' style='color: red'>Amlan Saha Kundu</a></p><h3>Help</h3><ul><li>Click ☰ on top left to open the MENU.</li><li>Click on your desired note to open.</li><li>To search a note, enter the name of the note in the search-bar</li><li>Click &#128376; on top right to open the Graph.</li><li>On 'Menu', toggle the switch to enable/ disable dark mode.</li></ul></div></div></div><div id='plotgraph'><a href='javascript:void()' id='closegraphbtn' onclick='closegraph(); setTimeout(location.reload.bind(location), 200);'>×</a><div id='GraphView'></div></div></div><img src='https://leancoding.co/DOYB8A' style='width:1px !important;'><script src='https://d3js.org/d3.v3.min.js'></script><script src='https://rawcdn.githack.com/yoursamlan/pubsidian/a7da9546987b84dfc80c52178f802e212d0a6ea8/CDN/network.js' type='text/javascript'></script><script src='https://rawcdn.githack.com/yoursamlan/pubsidian/a7da9546987b84dfc80c52178f802e212d0a6ea8/CDN/script.js' type='text/javascript'></script><script src='https://rawcdn.githack.com/yoursamlan/pubsidian/a7da9546987b84dfc80c52178f802e212d0a6ea8/CDN/data.js' type='text/javascript'></script><script src='https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML'></script></body></html>

And pages/nokia.html:
<p>
 a
</p>

**Comment by CatWithCode** [comment]
Here is everything. Its only a few lines of code in everything /Inside the TXT)
[Everthing.txt](https://github.com/yoursamlan/pubsidian/files/8312338/Everthing.txt)

**Comment by yoursamlan** [comment]
Works perfectly to me.
https://test-nokia.netlify.app/

Did you double-click `index.html` to open in chrome or host it locally ?

**Comment by CatWithCode** [comment]
ok. i uploaded it to my github and it themes to work. I actually did double-click the index.html. i expect it to behave like any other locally stored Wegpage but it themes like the website needs to be hosted to work. Thx!

I have few questions left:
1. How can i make a link to specific note? Stuff like "username.github.io/nokia" so that people directly get to the nokia note and not to the place where they last visit it. I want to use my notes as a github website so i need the ability to link to specific notes.
2. Am i allowed to change the welcome text? I want to add a few more details to it.
![image](https://user-images.githubusercontent.com/56090976/159242991-3c614948-9923-4aed-af07-e4810ac36c6d.png)

**Comment by yoursamlan** [comment]
I thought so 😅

Anyway, before answering your queries, I want to let you know how pubsidian works:
pubsidian retains its theme and the last opened note, based on two variables 
1. Content ID (CID) 
2. darkflag (1 for light, 0 for dark)

Initially, when someone opens your pubsidian (eg, test-nokia), the default CID value is 'X' and the darkflag is 1. The main loader of pubsidian will then load the website accordingly (eg: apply the light theme (darkflag=1) and show the 'Welcome to Pubsidian' text at the main div (CID=X). 

Now, when you click a different note, it will change the variables accordingly and next time it will load exactly where you left off
![image](https://user-images.githubusercontent.com/33586885/159339352-12ac5327-aae5-4a36-916f-8205f795e397.png)

Now answering your questions:
1. If you are using the current version, then you have to manually set the CID for that, you can use something like this...

```
<script>
let data = {CID:'x', darkflag:'1'};
localStorage.setItem('queryData', JSON.stringify(data));

function update(value){
    let prevData = JSON.parse(localStorage.getItem('queryData'));
    Object.keys(value).forEach(function(val, key){
         prevData[val] = value[val];
    })
    localStorage.setItem('queryData', JSON.stringify(prevData));
}
</script>
```
add this script inside <head> element and then use...
```
<body onload='update({CID: "4"})'>
```
 *change the CID value according to your note*
 
Since I have the exam in April, I'm unable to add an automated or easier approach right now, but in the next version (Which probably will appear in the first week of May), I will try to implement some solution to this problem, but for now, you have to change this manually as I have told you.


2. Yes you are free to change/modify anything. Just be careful, so that it doesn't affect the function of the site. 

Hope it will clear all your queries.
 Good luck :)

**Comment by CatWithCode** [comment]
A lot of thank yous for Such fast and good help!!! Have a nice day!
